go.mod

```diff
@@ -4,14 +4,12 @@ go 1.16
 
 require (
 	github.com/miekg/dns v1.1.29
-	github.com/onsi/ginkgo v1.12.0 // indirect
-	github.com/onsi/gomega v1.9.0 // indirect
 	github.com/prometheus/client_golang v1.5.1
 	github.com/prometheus/client_model v0.2.0
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
 	k8s.io/api v0.19.1
-	k8s.io/apimachinery v0.19.1
+	k8s.io/apimachinery v0.23.3
 	k8s.io/client-go v0.19.1
```

Comment:
you should bump this one too, to have all k8s libs from the same ver.

Reply:
@jsafrane As @aojea mentioned by seeing the [failing logs](https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/pull/120#issuecomment-1031926039) after bumping `k8s.io/client-go`, we need to new golang version for `io/fs` package but we don't have an idea where it is configured. 

Do you have any idea about this ? Please inform.